src/thread/thread_manager.cpp

```diff
@@ -421,14 +421,16 @@ namespace cubthread
     std::size_t max_active_workers = NUM_NON_SYSTEM_TRANS;  // one per each connection
     std::size_t max_conn_workers = NUM_NON_SYSTEM_TRANS;    // one per each connection
     std::size_t max_vacuum_workers = prm_get_integer_value (PRM_ID_VACUUM_WORKER_COUNT);
+    std::size_t max_server_type_workers =
+	    4; // used in page_server or tran_server. TODO set depending on server type and configurable.
```

Comment:
#4667 addresses this, and here or there the number for the added worker pool has to be taken into consideration as well. After the discussion on https://github.com/CUBRID/cubrid/pull/4667#discussion_r1319572377.